jaeger_client/thrift.py

```diff
@@ -13,37 +13,48 @@
 # limitations under the License.
 
 import traceback
```

Comment:
I thought your objective was to supply types for public APIs. Do we need to touch these internal files?

Reply:
Can you please tell me then, which files are considered "public"?

Reply:
Strictly speaking, the only APIs from Jaeger that users need to use directly are the config.py module and the Tracer constructor, all other integrations with the SDK should be via OpenTracing API. However, Tracer constructor takes a lot of other internal types like Sampler / Reporter, so that technically increases the scope of how much typing is needed to help users.

Reply:
So span_context and TUDPTransport can be considered internal?